src/HDF5.jl

```diff
@@ -1371,10 +1373,10 @@ function read{S<:String}(obj::DatasetOrAttribute, ::Type{Array{S}})
         ret = Array{S}(sz)
         if isvar
             # Variable-length
-            buf = Vector{Ptr{UInt8}}(len)
+            buf = Vector{Cstring}(len)
             h5t_set_size(memtype_id, H5T_VARIABLE)
             readarray(obj, memtype_id, buf)
-            # FIXME? Who owns the memory for each string? Will Julia free it?
+            # FIXME? Who owns the memory for each string? Julia v0.6+ won't free it.
```

Comment:
Just to double check, do you know whether this comment still applies with your fix? If not no problem just wanted to know if this `#FIXME` can be removed or not. Thanks.